hw_8_Python/main.py

```diff
@@ -0,0 +1,219 @@
+        file.close()
+
+    with open("files/Employees.txt", "r", encoding="utf-8") as file:
+        full_tab = file.read().strip().split("\n")
+        index_tab = []
+        for i in range(1, len(full_tab)):
+            start_index = 1
+            end_intdex = full_tab[i][start_index:].find('"') + 1
+            index_tab.append(int(full_tab[i][start_index:end_intdex]))
+        file.close()
+    with open("files/Employees.txt", "a", encoding="utf-8") as file:
+        flag_employee = False
+        flag = False
+        name_emp = input("Введите имя: ").title()
+        birthday_emp = input("Введите дату рождения: ")
+        phone_emp = input("Введите телефон: ")
+        post_emp = f'"{input("Введите должность: ").capitalize().strip()}"'
+        for i in range(len(index_tab) + 1):
+            if i not in index_tab:
+                index_tab.append(i)
+                break
+
+        for j in range(len(posts)):
+            if post_emp in posts[j]:
+                department_emp = departments[int(posts[j][len(posts[j]) - 2:len(posts[j]) - 1])]
+                flag = True
+                if flag == True:
+                    new_employee = f'"{i}", "{name_emp}", "{birthday_emp}", ' \
+                                   f'"{phone_emp}", {post_emp}, "{department_emp}"\n'
+                    file.writelines(new_employee)
+                    flag_employee = True
+                    print("Запись о сотруднике создана.")
+                    if flag_employee == True:
+                        break
+            else:
+                print("Вы ввели должность, которой не существует.")
+                break
+
+
+def delete_in_main_tab(x):
+    with open("files/Employees.txt", "r", encoding="utf=8") as file:
+        employees = list(file.read().split("\n"))
+        file.close()
+    with open("files/Employees.txt", "w", encoding="utf-8") as file:
+        for i in range(0, x):
+            file.write(f'{employees[i]}\n')
+        for i in range(x + 1, len(employees)):
+            file.write(f'{employees[i]}\n')
+        file.close()
+    print(f"Строка удалена {x}.")
+
+
+men_num = 10
+fmen_num = 11
+
+fmen_men_num = men_num + fmen_num
+
+command_gen_bd = "0"
+command_edit_main_tab = "1"
+command_add_in_main_tab = "2"
+command_delete_in_main_tab = "3"
+command_open_main_tab_in_notepad = "4"
+command_close = "5"
+
+while True:
+    print(f'''База данных компании "IT-комплект".
+                Чтобы сгенерировать БД введите команду: {command_gen_bd}.
+                Для редактирования главной таблицы введите команду: {command_edit_main_tab}.
+                Для добавления данных в главную таблицу введите команду: {command_add_in_main_tab}.
+                Для удаления данных из главной таблицы введите команду: {command_delete_in_main_tab}.
+                Для открытия всех таблиц в блокноте введите команду: {command_open_main_tab_in_notepad}
+                Чтобы прервать выполнение введите команду: {command_close}''')
```

Comment:
Выводе большой и имя очень красивый отступ.
```suggestion
    print(f'''\rБаза данных компании "IT-комплект".
                \rЧтобы сгенерировать БД введите команду: {command_gen_bd}.
                \rДля редактирования главной таблицы введите команду: {command_edit_main_tab}.
                \rДля добавления данных в главную таблицу введите команду: {command_add_in_main_tab}.
                \rДля удаления данных из главной таблицы введите команду: {command_delete_in_main_tab}.
                \rДля открытия всех таблиц в блокноте введите команду: {command_open_main_tab_in_notepad}
                \rЧтобы прервать выполнение введите команду: {command_close}''')
```
Так будет лучше